crates/isograph_schema/src/validate_schema.rs

```diff
@@ -386,6 +387,40 @@ pub enum ValidateSchemaError {
         argument_type: UnvalidatedTypeName,
     },
 
+    #[error("Expected input of type {expected_type}, found variable {variable_name} of type {variable_type}")]
```

Comment:
Can we add the argument_name here?
```
The argument `${argument_name}` has type `${expected_type}`, but the variable $${variable_name} has type `${variable_type}`
```

Also notice the backticks and double $

We're not very consistent about the backticks :/